lib/templates/ComponentTpl.js

```diff
@@ -6,10 +6,11 @@ class ComponentTpl {
    *
    * @param name - the component name
    */
-  constructor( name ) {
+  constructor( name, postfix ) {
     this.type = 'component';
     this.isDir = true;
-    this.name = name;
+    this.name = postfix ? `${name}.${postfix}` : name;
```

Comment:
What is the default postfix?

Reply:
Not have default postfix, because many developers not want it.

Reply:
OK, this will lead to a new version (i.e., breaking change). Please update the README file.

Reply:
Done

Reply:
Great. Final question, are you sure the convention around the Vue community is not adding postfix?

Reply:
Not everyone. And if used, they use screen/page and etc.